src/components/Menu/MenuItem.tsx

```diff
@@ -31,6 +31,9 @@ class MenuItem extends UIComponent<any, any> {
     /** Shorthand for primary content. */
     content: customPropTypes.contentShorthand,
 
+    /** A menu item can show it is currently unable to be interacted with. */
+    disabled: PropTypes.bool,
```

Comment:
this should be reflected in the props interface of MenuItem once merged with master

Reply:
added in the interface